tasks.py

```diff
@@ -298,3 +298,44 @@ def tests(context, name=NAME, image_ver=IMAGE_VER, local=INVOKE_LOCAL):
     pytest(context, name, image_ver, local)
 
     print("All tests have passed!")
+
+
+@task
+def html(context, sourcedir="docs/source", builddir="docs/build"):
+    """Creates html docs using sphinx-build command.
+
+    Args:
+        context (obj): Used to run specific commands
+        sourcedir (str, optional): Source directory for sphinx to use. Defaults to "source".
+        builddir (str, optional): Output directory for sphinx to use. Defaults to "build".
+    """
+    print("Building html documentation...")
+    clean_docs(context, builddir)
+    command = f"sphinx-build {sourcedir} {builddir}"
+    context.run(command)
+
+
+@task
+def api_doc(context, sourcedir="diffsync", output="docs/source/api"):
```

Comment:
Any .gitignore needed for docs/source/api?

Reply:
This was an old function I put in there prior to having the sphinx-apidoc done automatically in a sphinx callback. I've removed this function and source/api docs should be included.